gridstatus/ercot.py

```diff
@@ -2079,7 +2081,7 @@ def get_unplanned_resource_outages(self, date, verbose=False):
         """
         doc = self._get_document(
             report_type_id=UNPLANNED_RESOURCE_OUTAGES_REPORT_RTID,
-            date=date.normalize() + pd.DateOffset(days=3),
+            date=date.normalize(),
```

Comment:
would it make sense to switch to `@support_date_range(frequency=None)` and then use `published_before` and `published_after` like we do for `get_load_forecast`? that will more reliable iterate over all the publish times i think

Reply:
Okay, that makes sense. I've updated the code.